ecs/elections/exceptions.py

```diff
@@ -15,12 +15,12 @@ def __str__(self):
 
 class IncorrectPreferenceOrderException(Exception):
     def __str__(self):
-        return "Error of votes' preference order"
+        return "Error of votes preference order"
```

Comment:
I have problems with apostrof sign in tests, so I just deleted them… :wink: Doesn't matter that much